pkg/aws/v1alpha3/awscontrolplane/validate_awscontrolplane_test.go

```diff
@@ -114,7 +114,7 @@ func TestAZCount(t *testing.T) {
 				logger:                 microloggertest.New(),
 			}
 
-			admissionRequest, err := awsControlPlaneAdmissionRequest(tc.azs, "m4.xlarge", "100.0.0")
+			admissionRequest, err := awsControlPlaneAdmissionRequest(tc.azs, "m4.xlarge", "15.0.0")
```

Comment:
I had to change the release version for older tests because they use resources in default namespace